probatus/missing_values/imputation.py

```diff
@@ -0,0 +1,384 @@
+            clf, X, y, scoring=self.scorer.scorer, cv=self.cv, n_jobs=self.n_jobs
+        )
+
+        temp_results = {
+            "strategy": strategy,
+            f"{self.scorer.metric_name} score": np.round(
+                np.mean(imputation_results), 3
+            ),
+            f"{self.scorer.metric_name} std": np.round(np.std(imputation_results), 3),
+        }
+
+        return temp_results
+
+    def compute(self):
+        """
+        Checks if fit() method has been run and computes the DataFrame with results of imputation for each
+        strategy.
+
+        Returns:
+            (pd.DataFrame):
+                 DataFrame with results of imputation for each strategy.
+        """
+        self._check_if_fitted()
+        return self.report_df
+
+    def fit_compute(self, X, y, column_names=None):
+        """
+        Calculates the cross validated results for various imputation strategies.
+
+        Args:
+            X (pd.DataFrame):
+                input variables.
+
+            y (pd.Series):
+                target variable.
+
+            column_names (None, or list of str, optional):
+                List of feature names for the dataset.
+                If None, then column names from the X dataframe are used.
+
+        Returns:
+            (pd.DataFrame):
+                DataFrame with results of imputation for each strategy.
+
+        """
+        self.fit(X, y, column_names=column_names)
+        return self.compute()
+
+    def plot(self, show=True, **figure_kwargs):
+        """
+        Generates plot of the performance of various imputation strategies.
+
+        Args:
+            show (bool, optional):
+                If True, the plots are showed to the user, otherwise they are not shown. Not showing plot can be useful,
+                when you want to edit the returned axis, before showing it.
+
+            **figure_kwargs:
+                Keyword arguments that are passed to the plt.figure, at its initialization.
+
+        Returns:
+            (plt.axis):
+                Axis containing the performance plot.
+        """
+        plt.figure(**figure_kwargs)
+
+        imp_methods = list(self.report_df["strategy"])
+        performance = list(self.report_df[f"{self.scorer.metric_name} score"])
+        std_error = list(self.report_df[f"{self.scorer.metric_name} std"])
+
+        y_pos = [i for i, _ in enumerate(imp_methods)]
+        x_spacing = 0.01
+        y_spacing = 2 * x_spacing
+        plt.barh(
+            y_pos,
+            performance,
+            xerr=std_error,
+            align="center",
+            color=np.random.rand(len(performance), 3),
+        )
+
+        for index, value in enumerate(performance):
+            plt.text(value + x_spacing, index + y_spacing, str(value), rotation=45)
+        plt.yticks(y_pos, imp_methods)
+        plt.xlabel(f"Metric ({(self.scorer.metric_name).replace('_',' ').upper()})")
+        plt.title("Imputation Techniques")
```

Comment:
Let's set the title to imputation techniques comparison or something similar

Reply:
Done.